cli/README.md

```diff
@@ -39,7 +39,7 @@ You can use `ovsx` for downloading extensions from an Open VSX instance. Extensi
 
 Variants:
  * `ovsx get <extension>`
-   downloads an extension and saves it in a file as specified in its download URL (usually in the format `namespace.extension-version.vsix`) in the current working directory.
+   downloads an extension and saves it in a file as specified in its download URL (usually in the format `namespace.extension-version@target.vsix`) in the current working directory.
```

Comment:
Could we mention here that for most extension there will be no `@target` part? I know it will make the description quite a bit lengthier, but it is
1. optional
2. not used as much and could confuse some users. We can point to the VS Code docs about the feature [[1](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions)]